test_examples.sh

```diff
@@ -1,11 +1,25 @@
 #!/usr/bin/env bash
-SOURIR="../sourir"
+
+LONG=$1
+
+export SOURIR="../sourir"
+
+function ncores {
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    echo `sysctl -n hw.ncpu || echo 8`
```

Comment:
Is 8 a valid default choice? It seems that among the recent Intel processors, only the high performance ones come with "8" cores (4 physical cores with hyper-threading). "4" seems to be more common these days?